src/lib/sdk.ts

```diff
@@ -84,33 +87,55 @@ class FlutterSDK {
     }
   }
 
-  private async initDarkSdkHomeFromFlutter() {
+  private async initDarkSdkHomeFromFlutter(flutterLookup: string) {
     try {
-      let flutterPath = await which('flutter');
+      let flutterPath: string;
+
+      if (flutterLookup.length == 0) {
+        flutterPath = await which('flutter');
+      } else {
+        const { stdout } = await execCommand(flutterLookup);
+        flutterPath = stdout;
+        if (stdout.length == 0) {
+          throw new Error('flutter lookup returned empty string');
```

Comment:
It would be better to `log('flutter lookup returned empty string')` and `workspace.showMessage('xxx')` ?